conbench/app/__init__.py

```diff
@@ -1,7 +1,9 @@
+from typing import Callable
+
 import flask as f
 
 app = f.Blueprint("app", __name__)
-rule = app.add_url_rule
+rule: Callable = app.add_url_rule
```

Comment:
Callable is not a super useful annotation in general, do you know what types this Callable takes?

Reply:
see further above where I mention https://github.com/python/mypy/issues/6356

Reply:
(this is not meant to be a useful type annotation)